src/main/java/sevice/impl/TransactionToDataBaseImpl.java

```diff
@@ -0,0 +1,31 @@
+package sevice.impl;
+
+import java.util.List;
+import java.util.Map;
+import service.TransactionToDataBase;
+import strategy.TransactionStrategy;
+
+public class TransactionToDataBaseImpl implements TransactionToDataBase {
+    private static final int INDEX_OF_TRANSACTION = 0;
+    private static final int INDEX_OF_FRUIT = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+    private String spliterator;
+
+    public TransactionToDataBaseImpl(String spliterator) {
+        this.spliterator = spliterator;
+    }
+
+    @Override
+    public boolean transactionToDataBase(List<String> data,
+                                         Map<String, TransactionStrategy> strategyMap) {
```

Comment:
Interesting solution!
I'm not sure you can use a map here, but let the mentors guide you)

Reply:
It's better to handle the choice of operation in the strategy class, having a map to choose from, it's better not to handle this in the service class